### DISCUSSION TITLE
Command Line and no terminal display

### DISCUSSION THREAD

**Discussion by WirthmU** [nicolargo/glances#2077] [General]
I can export to a json-file using the following command:
`glances --export json --export-json-file /tmp/fitpc.json`

How would I change the command so that there is just the file saved without getting the terminal screen with all the statistics?

If I do:
`glances -w --disable-webui --export json --export-json-file /tmp/fitpc.json`

I get:
`Export is only available in standalone or client mode`

Running on Ubuntu

**Comment by nicolargo** [comment]
Use --quiet:

glances --export json --export-json-file /tmp/fitpc.json --quiet

**Reply by WirthmU** [reply]
Thank you for your quick reply. It works but the command line stays as it is instead of givng it back to a system prompt.

The idea would be to save the parameters every 10 minutes and then use these json-files to plot the data using cacti. Is there a way to use glance for this?

**Comment by nicolargo** [comment]
@WirthmU it is not a Glances feature but a Linux one:

```
nohup glances --export json --export-json-file /tmp/fitpc.json --quiet &
```

Then to stop the jobs, get the jobs ID (one in the following example):

```
jobs
[1]+  Running                 nohup glances --export json --export-json-file /tmp/fitpc.json --quiet &
```

Kill it:

```
kill %1
[1]+  Terminated              nohup glances --export json --export-json-file /tmp/fitpc.json --quiet
```

**Comment by nicolargo** [comment]
Additionally to change the refresh from 3 seconds to 10 minutes, use -t option

**Reply by WirthmU** [reply]
Would it be an idea to run glances in a crontab so that it is started when the PC starts up such as:
`@reboot glances --export json --export-json-file /tmp/fitpc.json --quiet -t 300`

Does the json-file get updated every 5 minutes this way?

**Reply by WirthmU** [reply]
I finally managed to make it to work running glances as a service:
This is the file: sudo nano /etc/systemd/system/glances.service
```
[Unit] 
Description=Glances 
After=network.target 

[Service] 
User=root 
ExecStart=/usr/local/bin/glances --export json --export-json-file /tmp/fitpc.json --quiet -t 300 
Restart=on-abort 
RemainAfterExit=yes 

[Install] 
WantedBy=multi-user.target 
```
